### DISCUSSION TITLE
Name generator

### DISCUSSION THREAD

**Discussion by corysimmons** [jln13x/ui.jln.dev#2] [Ideas]
I get stage fright when it asks me for a palette title. Could it just combine the primary and accent color, with [unique-names-generator](https://www.npmjs.com/package/unique-names-generator) to suggest title names like:

- Bright Blue Orange Mountainside
- Red Pink Tiger
- Gray White Jelly

**Comment by jln13x** [comment]
#6 let me know what you think @corysimmons :)
